src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.js

```diff
@@ -60,7 +60,7 @@ const ImageRenderer = (props) => {
                 >
                     {({show}) => (
                         <PressableWithoutFocus
-                            styles={[styles.noOutline, styles.alignItemsStart]}
+                            style={styles.noOutline}
```

Comment:
Invalid prop being passed here. The correct prop is `styles`.

Reply:
nevermind, @Nikhil-Vats handle this in their PR https://github.com/Expensify/App/pull/19545